sale_quotation_number/models/sale_order.py

```diff
@@ -15,7 +15,7 @@ def create(self, vals):
         if self.is_using_quotation_number(vals):
```

Comment:
This method should be converted with `@api.model_create_multi` and `vals_list` or you will see a WARNING in the log that suggests this.

Reply:
Hi @dariodelzozzo I've done the changes as per your feedback. Kindly review it.